src/rsa/padding.rs

```diff
@@ -38,8 +41,8 @@ impl RSAPadding {
         }
         out[2 + pad_len] = 0;
 
-        let (digest_prefix, digest_dst) = out[3 + pad_len..].split_at_mut(
-            self.digestinfo_prefix.len());
+        let (digest_prefix, digest_dst) =
+                out[3 + pad_len..].split_at_mut(self.digestinfo_prefix.len());
```

Comment:
What's the point of this change? If you think this new formatting is better, please split it into a separate commit that consists just of reformatting changes.